src/Sulu/Bundle/AdminBundle/Resources/js/services/Requester/tests/Requester.test.js

```diff
@@ -17,6 +17,7 @@ test('Should execute GET request and reject with response when the response cont
     expect(window.fetch).toBeCalledWith('/some-url', {
```

Comment:
can we add a test to check if the hooks are called? looks like this is not tested at the moment